src/web/src/alert_ack.ecpp

```diff
@@ -59,14 +59,19 @@ bool database_ready;
     if (!database_ready) {
         log_debug ("Database is not ready yet.");
         std::string err = TRANSLATE_ME ("Database is not ready yet, please try again after a while.");
+        if (request.getMethod () == "PUT")
+            log_info_audit ("Request PUT alert ack FAILED");
```

Comment:
Just a comment. On failure treatment, we invoke log_info_audit(). It should be log_error_audit().

Reply:
Updated, thx